# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/libsoup/newer-gnutls.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN libsoup-1.99.28.orig/libsoup/soup-gnutls.c libsoup-1.99.28/libsoup/soup-gnutls.c
--- libsoup-1.99.28.orig/libsoup/soup-gnutls.c	2004-03-24 16:48:24.273716344 -0500
+++ libsoup-1.99.28/libsoup/soup-gnutls.c	2004-03-24 16:48:36.334882768 -0500
@@ -77,7 +77,6 @@
 
 	if (status & GNUTLS_CERT_INVALID ||
 	    status & GNUTLS_CERT_NOT_TRUSTED ||
-	    status & GNUTLS_CERT_CORRUPTED ||
 	    status & GNUTLS_CERT_REVOKED)
 	{
 		g_warning ("The certificate is not trusted.");
